compiler-plugin/test-fixtures/org/jetbrains/kotlin/compiler/plugin/template/runners/AbstractJvmBoxTest.kt

```diff
@@ -43,6 +44,8 @@ open class AbstractJvmBoxTest : AbstractFirBlackBoxCodegenTestBase(FirParser.Lig
                 ::PluginAnnotationsProvider,
                 ::ExtensionRegistrarConfigurator
             )
+
+            useCustomRuntimeClasspathProviders(::PluginRuntimeAnnotationsProvider)
```

Comment:
Could you extract these configurations into an extension function and also use it in the diagnostic base test? While the runtime classpath isn't needed for diagnostic tests, it's good to have a single configuration pattern for base tests. [See here for an example.](https://github.com/bnorm/buildable/blob/main/compiler-plugin/test-fixtures/dev/bnorm/buildable/plugin/BuildableExtensionRegistrarConfigurator.kt#L15)